pipeline.yml

```diff
@@ -91,10 +91,10 @@ jobs:
   plan:
   - aggregate:
     - get: broker-src
-      passed: [unit-test-uaa-guard-service]
+      passed: [unit-test-uaa-guard-service, push-uaa-guard-service-staging]
```

Comment:
so getting rid of `unit-test-uaa-guard-service` in the prod deployment? and just keep `push-uaa-guard-service-staging` instead?